src/main/scala/github4s/Implicits.scala

```diff
@@ -0,0 +1,63 @@
+package github4s
+
+import cats.std.FutureInstances
+import cats.std.future._
+import cats.{ Monad, Id, Eval, MonadError }
+import github4s.free.interpreters.Interpreters
+import scala.concurrent.{ ExecutionContext, Future }
+
+object Implicits {
+
+  implicit val evalMonadError: MonadError[Eval, Throwable] = new MonadError[Eval, Throwable] {
+
+    override def pure[A](x: A): Eval[A] = Eval.now(x)
+
+    override def map[A, B](fa: Eval[A])(f: A ⇒ B): Eval[B] = fa.map(f)
+
+    override def flatMap[A, B](fa: Eval[A])(ff: A ⇒ Eval[B]): Eval[B] =
+      fa.flatMap(ff)
+
+    override def raiseError[A](e: Throwable): Eval[A] =
+      Eval.later({ throw e })
+
+    override def handleErrorWith[A](fa: Eval[A])(f: Throwable ⇒ Eval[A]): Eval[A] =
+      Eval.later({
+        try {
+          fa.value
+        } catch {
+          case e: Throwable ⇒ f(e).value
+        }
+      })
+  }
+
+  implicit def idMonadError(implicit I: Monad[Id]): MonadError[Id, Throwable] = new MonadError[Id, Throwable] {
```

Comment:
Have you tested I'd and task in a file where either one was required as implícits. @dialelo and I have experienced implícits ambiguity presumably because Task also conforms to Id given I'd is applicable to anything. Not sure we should provide this instance here. We may have to look into a LowerPriorityImplicits trait stack to avoid ambiguity. See also cats for examples of this.